test/lib.test.js

```diff
@@ -84,22 +84,22 @@ const testVectorsSuccess = [
       value: '50',
     },
     expected: {
-      maxFlowValue: '50',
-      transferStepsCount: 2,
+      maxFlowValue: '0',
+      transferStepsCount: 0,
```

Comment:
this is suspicious, no?
if the input file is the same it should fins at least one step

Reply:
yeah... I am not sure why I checked with the binary  without running with node and returned the same, I could not test it back with the old binary because it's not compatible with my computer